projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * WordPress.com Site Menu
+ *
+ * Add's a WordPress.com menu item to the admin menu linking back to the sites WordPress.com home page.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Add a WordPress.com menu item to the wp-admin sidebar menu.
+ */
+function wpcom_add_wpcom_menu_item() {
+	if ( function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() ) {
+		$domain = wp_parse_url( home_url(), PHP_URL_HOST );
+		add_menu_page(
+			esc_attr__( 'WordPress.com', 'jetpack-mu-wpcom' ),
+			esc_attr__( 'WordPress.com', 'jetpack-mu-wpcom' ),
+			'manage_options',
+			"https://wordpress.com/home/$domain",
+			null,
+			'dashicons-arrow-left-alt2',
+			0
+		);
+
+		// Position a separator below the WordPress.com menu item.
```

Comment:
This looks really confusing, but seems to work. Seems fine for now.

Reply:
We could just drop it in at position 1 and hope nothing else injects into position 0.